common/trainer/default_trainer.py

```diff
@@ -147,12 +148,19 @@ def train(self):
                     if self.lr_scheduler is not None:
                         self.lr_scheduler.step()
                     self.optimizer.zero_grad()
-                    if step % log_step == 0:
-                        logger.info(f"train epoch:[{idx}/{num_train_epochs}]\tstep:[{step}/{total_steps}]\tloss:{loss:.6f}\tppl:{math.exp(loss):.6f}\ttime:{time.time()-start:.6f}")
-                        report({"train_epoch": idx, "total_epochs": num_train_epochs, "train_step": step, "total_steps": min(max_train_step, total_steps) if max_train_step else total_steps})
```

Comment:
This line is for updating the progress bar on web ui, the key of this dict can't be modified because of https://github.com/intel/llm-on-ray/blob/f26343d14320f0be889f46c3d1703a743de82f57/ui/start_ui.py#L82.

Reply:
Thanks for the info. Let me double check how to update this. All the keys and values will also be shown in Tensorboard.